src/components/CheckoutFunnel/OrderConfirmation/OrderConfirmation.module.scss

```diff
@@ -113,3 +115,55 @@
     margin-bottom: 20px;
   }
 }
+
+@media (max-width: $tablet-s) {
```

Comment:
So far we've been using the $mobile-width variable for the mobile version (970px) to be in line with the rest of the shop, which seems to use that breakpoint as well.